holidayextras/ui-toolkit
# Creating a quote component

#### What does this PR do? (please provide any background)
- React Bootstrap doesn't have a Quote component so this PR creates on.
- Creates a `<blockquote>` and `<q>` element
- Adds all associated [Schema](http://schema.org/)
- Allows you to ass a Cite, Author and Role

#### What tests does this PR have?
- Mocha tests check for type, purpose, cite, author and role

#### How can be this tested?
- Check the code for coding anti-patterns
- Run `npm run test` to run all mocha tests
- Also run `grunt docs` and check the example on the UIToolkit docs

#### Screenshots / Screencast
<img width="856" alt="screen shot 2015-07-07 at 13 00 06" src="https://cloud.githubusercontent.com/assets/778942/8545666/b10c49ae-24a8-11e5-8d8a-310eedfc5798.png">

#### What gif best describes how you feel about this work?
![url](https://cloud.githubusercontent.com/assets/778942/8545694/e8e87ba4-24a8-11e5-9006-55b80f9da446.jpg)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
